File: app/src/main/java/br/com/brqtest/services/ApiCEPRetrofit.java
package br.com.brqtest.services;

import br.com.brqtest.model.Endereco;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface ApiCEPRetrofit {

    String BASE_URL = "https://viacep.com.br/ws/";


    @GET("{CEP}/json/")
    Call<Endereco> getEnderecoByCEP(@Path("CEP") String CEP);

}

File: app/src/main/java/br/com/brqtest/serviceskotlin/ApiCEPRetrofit.kt
package br.com.brqtest.serviceskotlin

import br.com.brqtest.modelkotlin.Endereco
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path


interface ApiCEPRetrofit {


    @GET("{CEP}/json/")
    fun getEnderecoByCEP(@Path("CEP") CEP: String): Call<Endereco>

    companion object {

        val BASE_URL = "https://viacep.com.br/ws/"
    }

}
